CHANGELOG.md

```diff
@@ -2,4 +2,10 @@
 
 ### Features
 
+#### ilab command redesign
+
+The ilab command redesign included in v0.17 introduces a new command structure that follows a resource group design. This means that commands that once were something like `ilab chat` now are `ilab model chat`. The new groups are model, data, taxonomy, and config. The commands that fall under these are all of the pre-existing `ilab` commands just now grouped by the resource which the command commonly deals with.
+
+The old command structure is still aliased to work but will be removed in a future release.
```

Comment:
I think it'd be good to give people a timeline for this. Let's coordinate during the community call.

Reply:
Slack thread if anyone wants to jump in: https://instruct-lab.slack.com/archives/C06TJP5V726/p1718180412862799

Reply:
@cdoern let's go ahead and target `0.19.0` for this?